src/include/cephfs/types.h

```diff
@@ -45,6 +45,13 @@
 #define CEPH_FS_ONDISK_MAGIC "ceph fs volume v011"
 #define MAX_MDS                   0x100
 
+#define CEPH_4M_BLOCK_SHIFT 22
+#define CEPH_4M_BLOCK_SIZE (1 << CEPH_4M_BLOCK_SHIFT) // 4MB
+#define CEPH_4K_BLOCK_SHIFT 12
+#define CEPH_4K_BLOCK_SIZE (1 << CEPH_4K_BLOCK_SHIFT) // 4KB
```

Comment:
this needs a bit of correction. CEPH_4M_BLOCK_SIZE and CEPH_4K_BLOCK_SIZE are both power of 2. The comment here as well in other places say (M/K)B ( i.e power of 10) but it should be (M/K)iB. Will add this correction in the PR i raise for the affected test cases due to this patch.